### PR TITLE
Adding app identity blob signing example.

### PR DESCRIPTION
**Pull request by jonparrott**

### CODE REVIEW

File: appengine/app_identity/signing/main.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    cert_seq = DerSequence()
+    cert_seq.decode(cert_der)
+    tbs_seq = DerSequence()
+    tbs_seq.decode(cert_seq[0])
+    public_key = RSA.importKey(tbs_seq[6])
+
+    signer = PKCS1_v1_5.new(public_key)
+    digest = SHA256.new(data)
+
+    if signer.verify(digest, signature):
```
Comment (waprin) [inline appengine/app_identity/signing/main.py]: could be return signer.verify(digest, signature)

File: appengine/app_identity/signing/main.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    for cert in public_certificates:
+        if verify_signature(data, signature, cert.x509_certificate_pem):
+            return True
+
+    return False
+
+
+class MainPage(webapp2.RequestHandler):
+    def get(self):
+
```
Comment (waprin) [inline appengine/app_identity/signing/main.py]: nathanielpick: spurious whitepsace

**Comment by waprin** [comment]
Two minor comments, otherwise LGTM.

File: appengine/app_identity/signing/main.py (hunk 1 above)
Comment (jonparrott) [inline appengine/app_identity/signing/main.py]: That's what I get for copy-pasta.

File: appengine/app_identity/signing/main.py (hunk 2 above)
Comment (jonparrott) [inline appengine/app_identity/signing/main.py]: I'll allow it.

**Pull request merged**